programs/bpf_loader/src/mem_pool.rs

```diff
@@ -0,0 +1,146 @@
+use {
```

Comment:
This entire file could go into the `memory-management` crate. loader-v4 and potential even the broader runtime could use it as well.

Reply:
I could do this, but I checked and literally nothing uses the memory-management crate today. Seems strange that we'd add a dependency on it _just_ for this?

Reply:
Hm, then you need to find a different place. `loader-v4` needs the RBPF vm stuff as well.

Reply:
I've now moved it to solana-program-runtime